### PR TITLE
add RFC 1 -- initial creation of the PROJ Project Steering Committee

### PR DESCRIPTION
**Pull request by hobu**

### CODE REVIEW

File: docs/source/community/rfc/rfc-1.rst [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+* Any change to committee membership (new members, removing inactive members)
+* Changes to project infrastructure (e.g. tool, location or substantive
+  configuration)
+* Anything that could cause backward compatibility issues.
+* Adding substantial amounts of new code.
+* Changing inter-subsystem APIs, or objects.
+* Issues of procedure.
+* When releases should take place.
+* Anything dealing with relationships with external entities such as OSGeo
+* Anything that might be controversial.</li>
```
Comment (rouault) [inline docs/source/community/rfc/rfc-1.rst]: </li> should be removed

File: docs/source/community/rfc/rfc-1.rst [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  that the PSC will be the entity largely responsible for creating any
+  such committees.
+
+Committee Membership
+---------------------
+
+The PSC is made up of individuals consisting of technical contributors
+(e.g. developers) and prominent members of the PROJ user community.
+There is no set number of members for the PSC although the initial desire
+is to set the membership at 9.
```
Comment (rouault) [inline docs/source/community/rfc/rfc-1.rst]: Should we keep this sentence about 9 members ? Since we are 6 active

**Review by rouault — APPROVED** [review]

**Comment by kbevers** [comment]
Should it be explained what it means to be "emeritus"? For instance, does emeritus status give you a stronger vote? A weaker vote? Can you vote at all? How do you obtain emeritus status? After having served as chair for a certain period? Or is this a one-off thing for Frank?
I reckon the intention is to give Frank a permanent seat in the PSC similar to Gary Sherman in the QGIS PCS. I am absolutely okay with that I just think it needs mentioning of some sort.

**Comment by hobu** [comment]
IMO, "emeritus" doesn't mean anything other than distinguished 😄. He votes and follows all the rules like any other member as far as I'm concerned. I made the distinction to attempt to direct the "hey you, PROJ person...." traffic toward you @kbevers

**Comment by kbevers** [comment]
> IMO, "emeritus" doesn't mean anything other than distinguished 😄. He votes and follows all the rules like any other member as far as I'm concerned. 

Alright. I think you should put that in somewhere then. To avoid confusion.

>  I made the distinction to attempt to direct the "hey you, PROJ person...." traffic toward you @kbevers

It seems to find me just fine already :)

**Pull request merged**